Wikipedia/Code/AccountViewController.swift

```diff
@@ -122,7 +122,13 @@ class AccountViewController: SubSettingsViewController {
                 
                 let loadingFlowController = TalkPageContainerViewController.talkPageContainer(title: title, siteURL: siteURL,  type: .user, dataStore: dataStore, theme: theme)
                 
-                self.navigationController?.pushViewController(loadingFlowController, animated: true)
+                let newTalkPage = TalkPageViewController(theme: theme)
+                if FeatureFlags.needsNewTalkPage {
+                    self.navigationController?.pushViewController(newTalkPage, animated: true)
+                } else {
+                    self.navigationController?.pushViewController(loadingFlowController, animated: true)
+                }
+                
```

Comment:
We should only instantiate the appropriate view controllers within the conditionals, so we're not spinning up a view controller that we'll never display:
```
if FeatureFlags.needsNewTalkPage {
    let newTalkPage = TalkPageViewController(theme: theme)
    self.navigationController?.pushViewController(newTalkPage, animated: true)
} else {
    let oldTalkPage = TalkPageContainerViewController.talkPageContainer(title: title, siteURL: siteURL,  type: .user, dataStore: dataStore, theme: theme)
    self.navigationController?.pushViewController(oldTalkPage, animated: true)
}
```